mmv1/templates/terraform/custom_flatten/float64_to_int_to_string.go.erb

```diff
@@ -0,0 +1,7 @@
+func flatten<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d *schema.ResourceData, config *Config) interface{} {
```

Comment:
What does this change, exactly? Why not just do a raw revert?

Reply:
mtu is broken today on `3.61.0` when reading `mtu` you will get the following error.

```
Error: Error reading InterconnectAttachment: mtu: '' expected type 'string', got unconvertible type 'float64'
```

example config used

```terraform

provider "google" {
  version = "3.61.0"
}

resource "google_compute_interconnect_attachment" "default" {
  name                     = "interconnect"
  router                   = google_compute_router.foobar.self_link
  edge_availability_domain = "AVAILABILITY_DOMAIN_1"
  type                     = "PARTNER"
  mtu = 1440
}

resource "google_compute_router" "foobar" {
  name    = "myrouter"
  network = google_compute_network.foobar.name
  bgp{
  asn = 16550
  }
}

resource "google_compute_network" "foobar" {
  name                    = "my-network2"
   auto_create_subnetworks = false
}
```